index.html

```diff
@@ -41,5 +41,163 @@ <h2>I'am a Software Developer</h2>
+                  </ul> 
+                  <a href="#">See project</a>
+        </article>
+        <article>
+            <h3>Profesional Art Printing Data</h3>
+
+            <p>A daily selection of privately personalized reads; no 
+                accounts or sign-ups required. has been the industry's
+                 standard</p>
+
+                 <ul>
+                    <li>html</li>
+                    <li>bootstrap</li>
+                    <li>Ruby</li>
+                  </ul> 
+                  <a href="#">See project</a>
+        </article>
+        <article>
+            <h3>Profesional Art Printing Data</h3>
+
+            <p>A daily selection of privately personalized reads; no 
+                accounts or sign-ups required. has been the industry's
+                 standard</p>
+
+                 <ul>
+                    <li>html</li>
+                    <li>bootstrap</li>
+                    <li>Ruby</li>
+                  </ul> 
+                  <a href="#">See project</a>
+        </article>
+        <article>
+            <h3>Profesional Art Printing Data</h3>
+
+            <p>A daily selection of privately personalized reads; no 
+                accounts or sign-ups required. has been the industry's
+                 standard</p>
+
+                 <ul>
+                    <li>html</li>
+                    <li>bootstrap</li>
+                    <li>Ruby</li>
+                  </ul> 
+                  <a href="#">See project</a>
+        </article>
+        <article>
+            <h3>Profesional Art Printing Data</h3>
+
+            <p>A daily selection of privately personalized reads; no 
+                accounts or sign-ups required. has been the industry's
+                 standard</p>
+
+                 <ul>
+                    <li>html</li>
+                    <li>bootstrap</li>
+                    <li>Ruby</li>
+                  </ul> 
+                  <a href="#">See project</a>
+        </article>
+        <article>
+            <h3>Profesional Art Printing Data</h3>
+
+            <p>A daily selection of privately personalized reads; no 
+                accounts or sign-ups required. has been the industry's
+                 standard</p>
+
+                 <ul>
+                    <li>html</li>
+                    <li>bootstrap</li>
+                    <li>Ruby</li>
+                  </ul> 
+
+                  <a href="#">See project</a>
+        </article>
+
+    </section>
+
+    <section class="about-container">
+        <h6>skills</h6>
+        <article>
+            <h2>About me</h2>
+            <p>Hello I'm a software developer! I can help you
+                 build a product , feature or website Look 
+                 through some of my work and experience! If you
+                  like what you see and have a project you need
+                   coded, don't hestiate to contact me.</p>
+
+                   <a href="#">See project</a>
```

Comment:
Kindly replace the text of the button to be the same as in Figma Design 🤗

_Your design_
![image](https://user-images.githubusercontent.com/73714615/185618465-918d622c-c879-437f-8e1a-c0ee6e23218d.png)

_Figma Design_
![image](https://user-images.githubusercontent.com/73714615/185618498-3c75a532-7f08-4163-b15b-086df9a9445a.png)